sagemaker-pipeline-parameterization/notebook.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>Grammar - pluralize - <code>Pipeline Parameters are input parameters</code></p><p> </p><p><code>S3 URI for to python script</code> - delete either "for" or "to"</p><p> </p><p>Be consistent ending list lines with or without a period.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>